lib/generators/pageflow/error_pages/templates/404.html

```diff
@@ -3,7 +3,7 @@
 <head>
   <meta charset="utf-8"/>
   <title>Beitrag nicht gefunden (404)</title>
```

Comment:
The generated default file should probably be in english as that's also the default locale.